tm-bench/transacter.go

```diff
@@ -186,16 +195,21 @@ func (t *transacter) sendLoop(connIndex int) {
 			}
 
 			timeToSend := time.Now().Sub(startTime)
+			logger.Info(fmt.Sprintf("sent %d transactions", numTxSent), "took", timeToSend)
 			if timeToSend < 1*time.Second {
-				time.Sleep(time.Second - timeToSend)
+				sleepTime := time.Second - timeToSend
+				logger.Info(fmt.Sprintf("connection #%d is sleeping for %f seconds", connIndex, sleepTime.Seconds()))
```

Comment:
This is going to generate a lot of output. Does the client need to know about sleeping?

Reply:
I actually found it pretty useful for debugging. (I found that the rate transactions are created is fast for the first block, but significantly slower in the second block until the end)

Perhaps this could be moved to logger.Debug? (Or the sleeping information can be consolidated into the already existent messages)

Reply:
`logger.Debug` is fine